inc/Engine/Preload/Subscriber.php

```diff
@@ -375,4 +387,20 @@ public function preload_after_automatic_cache_purge( $deleted ) {
 			}
 		}
 	}
+
+	/**
+	 * Add the excluded uri from the preload to the filter.
+	 *
+	 * @param array $regexes regexes containing excluded uris.
+	 * @return array|false
+	 */
+	public function add_preload_excluded_uri( $regexes ): array {
+		$preload_excluded_uri = $this->options->get( 'preload_excluded_uri', [] );
```

Comment:
```suggestion
		$preload_excluded_uri = (array) $this->options->get( 'preload_excluded_uri', [] );
```

Guard against failure in array_merge if that's not an array.